gazebo/sensors/SensorManager.cc

```diff
@@ -46,6 +50,42 @@ bool g_sensorsDirty = true;
 std::condition_variable
     SensorManager::ImageSensorContainer::conditionPrerendered;
 
+/// Performance metrics variables
+/// \brief last sensor measurement sim time
+std::map<std::string, gazebo::common::Time> sensorsLastMeasurementTime;
+
+/// \brief last sensor measurement real time
+std::map<std::string, gazebo::common::Time> worldLastMeasurementTime;
+
+/// \brief Data structure for storing metric data to be published
+struct sensorPerformanceMetricsType
+{
+  /// \brief sensor real time update rate
+  double sensorRealUpdateRate;
+
+  /// \brief sensor sim time update rate
+  double sensorSimUpdateRate;
+
+  /// \brief Rendering sensor average real time update rate
```

Comment:
Is it worth adding how this `sensorFPS` differs to `sensorRealUpdateRate`? Naively, the naming and this /brief make them sound the same except for one is an average.